StarryExpanse/Structs/InteractabilityPollResponse.h

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2017 59 Volt Entertainment, all rights reserved.
+//
+#pragma once
+
+#include "Actors/EInteractable.h"
+#include "CoreMinimal.h"
+#include "InteractabilityPollResponse.generated.h"
+
+USTRUCT(BlueprintType)
+struct FInteractabilityPollResponse {
```

Comment:
How about naming this FInteractabilityState? The "PollResponse" affectation communicates the method of delivery, which may change i the future. Unless that is a specific UE4 idiom then I'd keep it.

Reply:
Hmm the only thing is that it's not really a state, it could be based on a lot of things, calculated dynamically, etc. and sometimes it's just a redirector to do another query at a deeper level. I probably misnamed this thing when I said "Poll" because it's not exactly a poll as much as a "survey", but survey doesn't seem like the right word either.